### PR TITLE
Create the beginning of the game

### PR DESCRIPTION
**Pull request by yaboistyryl**
Created the beginning of the game

### CODE REVIEW

File: adventureGame/AdventureGame/.spyproject/workspace.ini [hunk 1]
```diff
@@ -6,5 +6,5 @@ save_non_project_files = False
 
 [main]
 version = 0.1.0
-recent_files = []
+recent_files = ['C:\\Users\\mr_ep\\.spyder-py3\\temp.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\obj\\Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\test\\test_Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\src\\Main.py']
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/.spyproject/workspace.ini:9]: way too many AdventureGame subfolders, lol

File: adventureGame/AdventureGame/src/Main.py [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+    print("| Enter any value to continue! |")
+    print("|==============================|")
+    input()
+    for i in range(50):
+        print("\n")
+    
+def introduction():
+    print("Sir Gregos IV: \"Quickly, in here! They're too many of them!\"\n")
+    time.sleep(3)
+    print("You and your friend hurry inside the ominous room where sir Gregos IV awaits, closing the door behind you.\n")
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/src/Main.py]: Change print statement to:
print("You and your friend hurry inside the ominous room where Sir Gregos IV awaits, close the door behind you.\n)

File: adventureGame/AdventureGame/.spyproject/workspace.ini (hunk 1 above)
Comment (yaboistyryl) [reply adventureGame/AdventureGame/.spyproject/workspace.ini:9]: Can't help this, it's auto-generated when you load the spyder project file

File: adventureGame/AdventureGame/src/Main.py [hunk 3]
```diff
@@ -28,17 +43,51 @@ def introduction():
     time.sleep(3)
     print("Sir Gregos IV: \"The names Sir Gregos IV, long serving commander of the king\'s forces\"\n")
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/src/Main.py]: Change print statement to:
print("Sir Gregos IV: \"The name's Sir Gregos IV, long serving commander of the king\'s forces\"\n")

File: adventureGame/AdventureGame/src/Main.py (hunk 2 above)
Comment (yaboistyryl) [reply adventureGame/AdventureGame/src/Main.py]: perhaps it should be: print("You and your friend hurry inside the ominous room where Sir Gregos IV awaits, and he closes the door behind you.\n) as what you said doesn't make a lot of sense to me

File: adventureGame/AdventureGame/src/Main.py (hunk 3 above)
Comment (yaboistyryl) [reply adventureGame/AdventureGame/src/Main.py]: Fixed.

**Review by Aymanchoudhery — APPROVED** [review]
Code is clean to follow and easy to read. I only noticed three minor grammatical errors; those aside, everything meets our criteria.

File: adventureGame/AdventureGame/src/Main.py (hunk 2 above)
Comment (Aymanchoudhery) [reply adventureGame/AdventureGame/src/Main.py]: My mistake. I mistook the third person text as speech from Sir Gregos. your original print statement was fine.

File: adventureGame/AdventureGame/.spyproject/workspace.ini (hunk 1 above)
Comment (Aymanchoudhery) [reply adventureGame/AdventureGame/.spyproject/workspace.ini:9]: No worries, would be confusing if it was displayed like that on your PC is all.

**Pull request merged**